README.md

```diff
@@ -34,35 +32,35 @@ There are two projects in this sample.  Each needs to be separately registered i
-6.Find the Application ID value and copy it to the clipboard.
+1. Sign in to the [Azure portal](https://portal.azure.com).
+2. On the top bar, click on your account and under the **Directory** list, choose the Active Directory tenant where you wish to register your application.
+2. Click on **More Services** in the left hand nav, and choose **Azure Active Directory**.
+3. Click on **App registrations** and choose **Add**.
+4. Enter a friendly name for the application, for example 'TodoListService' and select 'Web Application and/or Web API' as the Application Type. For the sign-on URL, enter the base URL for the sample, which is by default `https://localhost:44321`. For the App ID URI, enter `https://<your_tenant_name>/TodoListService`, replacing `<your_tenant_name>` with the name of your Azure AD tenant. Click on **Create** to create the application.
+5. While still in the Azure portal, choose your application, click on **Settings** and choose **Properties**.
+6. Find the Application ID value and copy it to the clipboard.
 
 #### Register the TodoListDaemonWithCert app
 
 
-1.Sign in to the [Azure portal](https://portal.azure.com).
-2.On the top bar, click on your account and under the **Directory** list, choose the Active Directory tenant where you wish to register your application.
-2.Click on **More Services** in the left hand nav, and choose **Azure Active Directory**.
-3.Click on **App registrations** and choose **Add**.
-4.Enter a friendly name for the application, for example 'TodoListDaemonWithCert' and select 'Web Application and/or Web API' as the Application Type. 
-5.Since this application is a daemon and not a web application, it doesn't have a sign-in URL or app ID URI.  For these two fields, enter "http://TodoListDaemonWithCert". Click on **Create** to create the application.
-6.While still in the Azure portal, choose your application, click on **Settings** and choose **Properties**.
-7.Find the Application ID value and copy it to the clipboard.
+1. Sign in to the [Azure portal](https://portal.azure.com).
+2. On the top bar, click on your account and under the **Directory** list, choose the Active Directory tenant where you wish to register your application.
+2. Click on **More Services** in the left hand nav, and choose **Azure Active Directory**.
+3. Click on **App registrations** and choose **Add**.
+4. Enter a friendly name for the application, for example 'TodoListDaemonWithCert' and select *'Web Application and/or Web API'* as the Application Type (even if here we have a console application). 
+5. Since this application is a daemon and not a web application, it doesn't have a sign-in URL or app ID URI.  For these two fields, enter "http://TodoListDaemonWithCert". Click on **Create** to create the application.
+6. While still in the Azure portal, choose your application, click on **Settings** and choose **Properties**.
+7. Find the Application ID value and copy it to the clipboard.

```

Comment:
The app ID is not going to be used anytime soon so there's no point in saving it to the clipboard.